### ISSUE TITLE
Question about StackRefill

### ISSUE THREAD

**Issue by f0kx** [Serilum/.issue-tracker#394]
https://imgur.com/a/FnLQuos

What is this Error about?

**Comment by James103** [comment]
Can you try posting (the contents of) `crash-2021-01-27_05.54.03-fml.txt` here?

**Comment by f0kx** [comment]
sry you can close this because iam dumb :D 
i was using 1.6.5 because i downloaded the latest version and i thought becaue many mods have 1.16.2/3/4/5 compatibility that it was not working!

**Comment by TheIrishStud** [comment]
If you're looking for fishing rod refill compatibility on 1.16.4 there is no version compatible yet... This is the same crash I got when trying to play 1.16.4 with shaders while I waited for 1.16.5 Forge to get updated enough to support shaders. If a 1.16.4 version doesn't get uploaded within the next week I think it would be safe to assume 1.16.4 will never have fishing rod stack refill support. I'm not part of the team or anything so take this last bit with a grain of salt... It's just my opinion after all.
